### PR TITLE
Consistent code formatting

### PR DESCRIPTION
**Pull request by 12awsomeman34**
This generally puts the code formatting into a more readable (imo) format. In the current docs, certain code blocks would get cut off into a new line due to being too long. With this, the code is placed on a new line and indented for visibility (sorta similarly to how I've been writing docs for the past while). However I'm not too sure if this makes it more visible to 'everyone', or in fact makes the docs code examples less readable.
Thus input on this idea is appreciated. Note that I haven't bothered with the rest of them, just wanted general feedback before going through all of them.

You can compare the code examples by looking at the Spongie preview and comparing that with current docs.

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/7ccf6055/index.html.

Here are some links to the pages that were modified:

- modified https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/7ccf6055/plugin/commands/arguments.html
- modified https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/7ccf6055/plugin/commands/commandcallable.html
- modified https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/7ccf6055/plugin/commands/creating.html
- modified https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/7ccf6055/plugin/plugin-class.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/496#issuecomment-220760092), not to the direct url to the preview._

File: source/plugin/commands/arguments.rst [hunk 1]
```diff
@@ -51,13 +51,13 @@ Example: Building a Command with Multiple Arguments
 
             .executor(new CommandExecutor() {
                 @Override
-                public CommandResult execute(CommandSource src, CommandContext args) throws CommandException {
-
+                public CommandResult execute(CommandSource src, CommandContext args)
+                                                                  throws CommandException {
```
Comment (Minecrell) [inline source/plugin/commands/arguments.rst]: IMO you should use the Sponge code style that would intend this only 4 spaces after `public` (or 8? Can't remember right now)

File: source/plugin/commands/arguments.rst [hunk 2]
```diff
@@ -51,13 +51,13 @@ Example: Building a Command with Multiple Arguments
...
                     Player player = args.<Player>getOne("player").get();
                     String message = args.<String>getOne("message").get();
-
+
```
Comment (Minecrell) [inline source/plugin/commands/arguments.rst]: Is there a reason you added trailing whitespace here?

File: source/plugin/commands/arguments.rst (hunk 1 above)
Comment (12AwsomeMan34) [inline source/plugin/commands/arguments.rst]: It's four spaces.

File: source/plugin/commands/arguments.rst (hunk 2 above)
Comment (12AwsomeMan34) [inline source/plugin/commands/arguments.rst]: Atom likely.

File: source/plugin/commands/arguments.rst (hunk 1 above)
Comment (Minecrell) [inline source/plugin/commands/arguments.rst]: No, on this line there are 66 spaces :P

Comment (12AwsomeMan34) [inline source/plugin/commands/arguments.rst]: I meant the code style was four spaces. I'll probably change it to that.

**Comment by ST-DDT** [comment]
@12AwsomeMan34 Could you fix the outlined issues? Then we could those as examples for future code examples in the docs as reference.

Which line length did you use for wrapping the code examples?

**Comment by saladoc** [comment]
@12AwsomeMan34 are you going to finish this or shall I take a look?

**Comment by 12AwsomeMan34** [comment]
@saladoc Yeah, I'll get to it sometime soon.

**Comment by 12AwsomeMan34** [comment]
I'll open a second PR for this soon.

**Comment by ST-DDT** [comment]
Can the associated branch be deleted?

**Pull request closed without merging**